### PR TITLE
Enable SwarmMode for api model 2017-01-31

### PR DESCRIPTION
**Pull request by JiangtianLi**
This PR enables SwarmMode for api model 2017-01-31 in ACS Engine. Subsequent changes will happen in az cli and RP.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/acs-engine/730)

### CODE REVIEW

File: pkg/api/v20170131/validate.go [hunk 1]
```diff
@@ -98,6 +99,7 @@ func (a *Properties) Validate() error {
 		if agentPoolProfile.OSType == Windows {
 			switch a.OrchestratorProfile.OrchestratorType {
 			case Swarm:
+			case SwarmMode:
```
Comment (amanohar) [inline pkg/api/v20170131/validate.go]: I thought we were disabling support for SwarmMode Windows since the template is not ready.

Comment (JiangtianLi) [inline pkg/api/v20170131/validate.go]: Good catch. I misunderstood and thought we'd disable in RP. Will remove.

**Comment by JiangtianLi** [comment]
@acs-bot test this please

**Comment by JiangtianLi** [comment]
@amanohar I've updated the PR. Could you please review?

**Pull request merged**